components/tidb_query/src/codec/row_codec.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
I prefer to create a new directory called `row` and create a file named `v2` inside. (We shall later move current row format to `v1`).